assets/js/modules/analytics/components/module/ModuleOverviewWidget/SiteStats.js

```diff
@@ -0,0 +1,160 @@
+/**
+ * SiteStats component the ModuleOverviewWidget widget.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { CORE_USER } from '../../../../../googlesitekit/datastore/user/constants';
+import { extractAnalyticsDashboardData } from '../../../util';
+import { getCurrentDateRangeDayCount } from '../../../../../util/date-range';
+import GoogleChart from '../../../../../components/GoogleChart';
+import PreviewBlock from '../../../../../components/PreviewBlock';
+import { Cell, Row, Grid } from '../../../../../material-components';
+const { useSelect } = Data;
+
+export default function SiteStats( { loaded, selectedStat, report } ) {
+	const dateRange = useSelect( ( select ) => select( CORE_USER ).getDateRange() );
+
+	if ( ! loaded ) {
+		return (
+			<PreviewBlock width="100%" height="270px" />
+		);
+	}
+
+	const currentDayCount = getCurrentDateRangeDayCount( dateRange );
```

Comment:
See above, there is a dedicated selector for this.

Reply:
Updated.